tftest.py

```diff
@@ -0,0 +1,14 @@
+import gensim,os,time 
```

Comment:
This file is meant only for bench-marking the CPU vs GPU performance. Should I format it and move it to the scripts folder?